### ISSUE TITLE
va-file-input:  Error message & file size is skipped when navigating in browse mode - JAWS & Chrome

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/vets-design-system-documentation#4723]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

When using JAWS/Chrome in browse mode the error message text and file size is not reachable.
This is not a problem on NVDA/Firefox & VoiceOver/Safari.

## What I expected to happen
The error message text and file size text should be able to be navigated to in browse mode.

## Reproducing

- CSS Library version: Current
- Device: (e.g. iPhone 8, Macbook) JAWS/Chrome
- Browser: (e.g. Firefox, IE 11)

Steps to reproduce:

1. Go to https://design.va.gov/storybook/?path=/story/uswds-va-file-input--error-message
2. Upload a file
3. Use JAWS/Chrome
4. "Display a helpful error message" and the file size text are not reachable in JAWS browse mode.

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

**Comment by caw310** [comment]
Closing as this is a duplicate of https://github.com/department-of-veterans-affairs/va.gov-team/issues/123070
